spec/services/follow_service_spec.rb

```diff
@@ -136,6 +136,16 @@
         expect(Follow.find_by(account: sender, target_account: bob)&.languages).to match_array %w(en es)
       end
     end
+
+    describe 'cannot follow own account' do
+      it 'raises an exception and does not create a following relation' do
+        expect { subject.call(sender, sender) }
+          .to raise_error(FollowService::SelfFollowError)
+          .and(not_change { sender.following?(sender) })
+
+        expect(sender.following?(sender)).to be false
```

Comment:
This could be an `.from(false)` added to the end of the `not_change` above?

Reply:
Hmmm... I can try it out and see 🙂